A72-open-telemetry-tracing.md

```diff
@@ -36,98 +38,125 @@ and OpenCensus's binary format for (de)serialization. The header name is unique
+stream tracer to implement the feature, the same as Census.
+
+The OpenTelemetry API will coexist with the OpenCensus API. Only one
+`grpc-trace-bin` header will be sent for a single RPC as long as only one of
+OpenTelemetry or OpenCensus is enabled for the channel. The APIs to enable and
+configure OpenTelemetry tracing are different among languages due to different 
+underlying infrastructures.
+
+#### Java
+In Java, it will be part of global interceptors, so that the interceptors are
+managed in a more sustainable way and user-friendly. As a prerequisite, the stream
+tracer factory API will be stabilized. OpenTelemetryModule will be created with
+an OpenTelemetry API instance passing in for necessary configurations.
+Users can also rely on SDK autoconfig extension that configures the sdk object
+through environmental variables or Java system properties, then pass the 
+obtained sdk object to gRPC.
+
+```Java
+// Construct OpenTelemetry to be passed to gRPC OpenTelemetry module for 
+// trace and metrics configurations.
+SdkTracerProvider sdkTracerProvider = SdkTracerProvider
+  .builder()
+  .addSpanProcessor(
+     BatchSpanProcessor.builder(exporter).build())
+  .build();
+
+OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+  .setTracerProvider(sdkTracerProvider)
+  .setMeterProvider(...)
+  .setPropagators(
+     ContextPropagators.create(GrpcTraceBinPropagator.getInstance()))
+  .build();
+
+// Alternatively, use auto configuration:
+// OpenTelemetry openTelemetry = 
+// AutoConfiguredOpenTelemetrySdk.getOpenTelemetrySdk().
+
+// Create a module that hosts tracing infrastructures. Should document that 
+// the module implementation may change as OpenTelemetry evolves.
+OpenTelemetryModule otModule = OpenTelemetryModule.newBuilder().sdk(openTelemetry).build();
+
+GlobalInterceptors.setInterceptors(
+    Arrays.asList(otModule.getClientTracingInterceptor()),
+    Arrays.asList(otModule.getServerTracerFactory()));
+
+```
+
+#### C++
+The following new methods will be added in `OpenTelemetryPluginBuilder`.
+
+```C++
+class OpenTelemetryPluginBuilder {
+ public:
+  // If `SetTracerProvider()` is not called, no traces are collected.
+  OpenTelemetryPluginBuilder& SetTracerProvider(
+      std::shared_ptr<opentelemetry::metrics::TracerProvider> tracer_provider);
+  // Set one or multiple text map propagators for span context propagation, e.g.
+  // GrpcTraceBinTextMapPropagator or community standard ones like W3C, etc.
+  OpenTelemetryPluginBuilder& SetTextMapPropagator(
+      std::unique_ptr<opentelemetry::context::propagation::TextMapPropagator>
+          text_map_propagator);
+};
+```
+
+#### Go
+In Go, the OpenTelemetry stream tracers and interceptors will be provided for users to install.
+
+TODO: add Go API.
+
+### Tracing Information
+With the new OpenTelemetry plugin we will produce the same tracing information as we 
+produce for Census. The following tracing information during an RPC lifecycle 
```

Comment:
Please describe this from scratch without referring to OC.  For example, we should have a brief description of the difference between the parent call span vs. the attempt span on the client side.